src/protos/io/defang/v1/fabric.proto

```diff
@@ -690,3 +693,26 @@ message WhoAmIResponse {
   SubscriptionTier tier = 5;
   bool training_opt_out = 6;
 }
+
+message EstimateRequest {
+  Provider provider = 1;
+  string region = 2;
+  bytes pulumi_preview = 3;
+  DeploymentMode mode = 4;
+}
+
+message EstimateLineItem {
+  string description = 1;
+  string unit = 2;
+  float quantity = 3;
+  float cost = 4;
+  string currency = 5;
+}
+
+message EstimateResponse {
+  Provider provider = 1;
+  string region = 2;
+  float subtotal = 3;
+  string currency = 4;
```

Comment:
Let’s check the money type, because it’s a change that will be harder to do later